tests/vmi_lifecycle_test.go

```diff
@@ -220,8 +221,15 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 				Should(ContainSubstring(`"subcomponent":"libvirt"`))
 		})
 
-		DescribeTable("log libvirtd debug logs should be", func(vmiLabels, vmiAnnotations map[string]string, expectDebugLogs bool) {
-			options := []libvmi.Option{libvmi.WithResourceMemory("32Mi")}
+		DescribeTable("log libvirtd debug logs should be", decorators.WgS390x, func(vmiLabels, vmiAnnotations map[string]string, expectDebugLogs bool) {
+			arch := util.TranslateBuildArch()
+			var memory string
```

Comment:
supernit 
```
memory := "32Mi"
if arch == "s390x" {
    memory = "128Mi"
}
```

Reply:
comment not applicable after reverting the changes.